src/app/components/autocomplete/autocomplete.component.ts

```diff
@@ -29,10 +29,10 @@ export interface State {
 })
 export class AutocompleteComponent implements OnInit {
```

Comment:
Any specific reason autocomplete and card files were added here? (mainly asking because I don't think we'll get much re-use from our existing pages due to how the new site will be structured)

Reply:
The were added with the initial site creation to test lazy loading. They'll be removed when I have other components to replace them.